src/source/Ice/Network.c

```diff
@@ -148,25 +148,18 @@ STATUS getLocalhostIpAddresses(PKvsIpAddress destIpList, PUINT32 pDestIpListLen,
     return retStatus;
 }
 
-STATUS createSocket(PKvsIpAddress pHostIpAddress, PKvsIpAddress pPeerAddress, KVS_SOCKET_PROTOCOL protocol, UINT32 sendBufSize, PINT32 pSockFd)
+STATUS createSocket(UINT16 familyType, KVS_SOCKET_PROTOCOL protocol, UINT32 sendBufSize, PINT32 pOutSockFd)
```

Comment:
Not sure whether familyType should be UINT16 or the KVS_IP_FAMILY_TYPE enum type? Same with KvsIpAddress.family member. This is not a major issue for this change though

Reply:
changed to KVS_IP_FAMILY_TYPE